aidial_adapter_openai/utils/chat_completion_response.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Iterable, Literal, Self
+
+from aidial_sdk.utils.merge_chunks import merge_chat_completion_chunks
+from pydantic import BaseModel
+
+
+class ChatCompletionResponse(BaseModel):
```

Comment:
You can make this class generic, and use types from openai library, so instead of 
`resp: dict` you would get 

```python
class ChatCompletionResponse(BaseModel, Generic[T]):
   resp: T


class ChatComplationBlock(ChatCompletionResponse[ChatCompletionChunk]):
    ...

...
```